consentconstants/purposes.go

```diff
@@ -39,3 +40,87 @@ const (
 	// for you in other contexts, i.e. on other service, such as websites or apps, over time.
 	Measurement Purpose = 5
 )
+
+// TCF 2.0 Purposes
+const (
+	// Cookies, device identifiers, or other information can be stored or accessed on your device for the purposes presented to you.
```

Comment:
Are the "vendors can" and "vendors cannot" copied from the IAB spec?

Reply:
Yes they are ... specifically the definitions found in the GVL